README.md

```diff
@@ -110,20 +110,25 @@ _Note for Windows Users: You should use a [Bash Shell][bash] to follow the steps
 To compile Kelp from source:
 
 1. [Download][golang-download] and [setup][golang-setup] Golang _v1.13 or later_.
+    * Confirm that `$GOPATH` is set, and that `GOBIN=$GOPATH/bin`
 2. [Install Glide][glide-install] for dependency management
     * `curl https://glide.sh/get | sh`
 3. Install [Yarn][yarn-install] and [NodeJs][nodejs-install] to build the Kelp GUI
 4. Clone the repo into `$GOPATH/src/github.com/stellar/kelp`:
     * `git clone git@github.com:stellar/kelp.git`
-5. Change to the kelp directory and install the dependencies:
+    * Change to `kelp` directory for all following commands: `cd $GOPATH/src/github.com/stellar/kelp`
+5. Install the dependencies:
     * `glide install`
 6. Install the [astilectron-bundler][astilectron-bundler] binary in a folder that is in your `PATH`
+    * `go install` will install this by default to `$GOBIN`, so check that that folder's full filepath is in `$PATH`
```

Comment:
this is a little confusing since it's hard to tell which folder should be in the `$PATH`. This could refer to $GOBIN (intended) or to the source code directory of astilectron-bundler:
```
... check that that folder's full filepath is in $PATH
```

The above setup step is part of setting up Golang. With that in mind, would it make sense to instead say in step 1 above: `..., and that $PATH includes the full filepath of $GOBIN` ?
If we add this in step 1 then here in step 6 we can say:
```
use `go install` to install the astilectron-bundler binary into $GOBIN
```